src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,42 @@
+package calculator
+
+object Calculator {
+
+    private const val DELIMITER = " "
+
+    fun execute(expression: String?): Double {
+        require(expression != null && expression.isNotEmpty())
```

Comment:
`String?`에 대해 호출할 수 있는 `isNullOrEmpty()` 또는 `isNullOrBlank()`가 있습니다.

```suggestion
        require(expression.isNullOrBlank())
```